ods_ci/tests/Tests/650__distributed_workloads/test-run-kuberay-e2e.robot

```diff
@@ -30,6 +30,27 @@ Run TestRayJobSuspend test
     ...     DistributedWorkloads
     ...     Kuberay
     Run Kuberay E2E Test    TestRayJobSuspend
+
+Run TestRayJobLightWeightMode test
```

Comment:
@jiripetrlik This test failed for me when I ran it on OCP.
Didi it pass for you?

Reply:
This test also failed for me